src/Parser/Druid/Balance/Modules/Items/OnethsIntuition.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SpellIcon from 'common/SpellIcon';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Wrapper from 'common/Wrapper';
+
+class OnethsIntuition extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  freeStarsurgeProcs = 0;
+  freeStarfallProcs = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasWrists(ITEMS.ONETHS_INTUITION.id);
+  }
+
+  on_toPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.ONETHS_INTUITION.id) {
+      this.freeStarsurgeProcs += 1;
+    }
+    if (spellId === SPELLS.ONETHS_OVERCONFIDENCE.id) {
+      this.freeStarfallProcs += 1;
+    }
+  }
+
+  item() {
+    return {
+      item: ITEMS.ONETHS_INTUITION,
+      result: (
+          <Wrapper>{this.freeStarsurgeProcs} <SpellIcon id={SPELLS.ONETHS_INTUITION.id}/> {this.freeStarfallProcs} <SpellIcon id={SPELLS.ONETHS_OVERCONFIDENCE.id}/></Wrapper>
```

Comment:
For better display, you could try something like the stats box in Frost Mage's Winter's Chill module (https://github.com/WoWAnalyzer/WoWAnalyzer/blob/b568ef41a82cc30e4c96f129f44b53e3af63a971/src/Parser/Mage/Frost/Modules/Features/WintersChill.js#L171)

Also, it would be nice to include a tooltip to more clearly explain to the user what the symbols mean, as they might not recognize them by icon.

Finally, as "extra credit", it would also be nice to show the player how much Astral Power this leggo saved them.

Reply:
It would definitely look nicer as a stat box, I just avoided it since it's an item. Probably makes more sense as a stat box and then throwing the procs/casts as a percent on the item.  I don't think it should display Astral Power saved as you usually wouldn't be casting the opposite spender without the proc, meaning the proc is worth less than the AsP it saves.

Reply:
No, keep it in Items... I guess you would need to modify more about that code in order to get it to display in Item format what you want. Really, I think the small icons are ok as long as you include a tooltip saying what's going on.

As for the AsP bit, it's a good point I wasn't thinking about how you might Starfall when you otherwise wouldn't.